src/libraries/System.Data.SqlClient/tests/ManualTests/SQL/ConnectionPoolTest/ConnectionPoolTest.cs

```diff
@@ -76,9 +77,9 @@ private static void BasicConnectionPoolingTest(string connectionString)
         /// Tests if killing the connection using the InternalConnectionWrapper is working
         /// </summary>
         /// <param name="connectionString"></param>
+        [Conditional("DEBUG")]
```

Comment:
This is just a cosmetic change. I would avoid any unnecessary churn under System.Data.SqlClient. The live copy of this code is in https://github.com/dotnet/SqlClient repo.